README.md

```diff
@@ -220,6 +220,35 @@ envsubst < deployment/private-installer.yaml | kubectl apply -f -
 
```

Comment:
To keep our commit history clean, we prefer if you squash all your commits into a single commit before opening the PR.

In this scenario, the commit message of the single squashed commit would just be the title of this PR: "make throttling params qps and burst configurable"